src/services/auth-services.ts

```diff
@@ -41,29 +41,34 @@ export default class AuthServices {
    * Fetches affiliation invites tied to this entity.
    * @param authApiUrl
    * @param businessId the business identifier (aka entity inc no)
+   * @param orgId org which has access rights to display (current logged in org)
    * @returns the axios response
    */
-  static async fetchAffiliationInvitations (authApiUrl: string, businessId: string) {
-    const url = `${authApiUrl}entity/${businessId}/affiliations/invitations`
-    return axios.get(url)
+  static async fetchAffiliationInvitations (authApiUrl: string, businessId: string, orgId: number) {
+    const url = `${authApiUrl}affiliationInvitations`
+    return axios.get(url, { params: { toOrgId: orgId, businessIdentifier: businessId, statuses: 'PENDING' } })
   }
 
   /**
    * Authorizes or refuses authorization for this invitation.
    * @param authApiUrl
-   * @param businessId the business identifier (aka entity inc no)
    * @param affiliationInvitationId id of affiliation to approve or not
    * @param isAuthorized boolean stating if invitation is authorized (true) or not authorized (false)
    * @returns the axios response
    */
   static async authorizeAffiliationInvitation (
     authApiUrl: string,
-    businessId: string,
     affiliationInvitationId: number,
     isAuthorized: boolean) {
+    let authorizePath = `affiliationInvitations/${affiliationInvitationId}/authorization/refuse`
```

Comment:
Minor: I think this would be clearer if you set a variable like `const action = isAuthorized ? 'accept' : 'refuse'` and then just set the path = `...${action}`